server/src/domain/library/library.service.ts

```diff
@@ -69,6 +78,106 @@ export class LibraryService {
     this.configCore.config$.subscribe((config) => {
       this.jobRepository.updateCronJob('libraryScan', config.library.scan.cronExpression, config.library.scan.enabled);
     });
+
+    if (this.watchLibraries) {
+      const libraries = await this.repository.getAll(false, LibraryType.EXTERNAL);
+
+      for (const library of libraries) {
+        await this.watch(library.id);
+      }
+    }
+  }
+
+  async watch(id: string): Promise<boolean> {
```

Comment:
Did I miss something or can this method be private? 🤔

Reply:
True, but the tests call it directly so I'm not sure what would be best

Reply:
We should really try to minimize the amount of code changes we have just to make testing work. E2e tests should be "black box" to begin with, exposing internal methods just to make them work is probably a sign that something else is wrong and needs to change.

Reply:
Done, had to rewrite a number of tests but watch() is now private